backend/package.json

```
@@ -1,67 +1,14 @@
 {
-  "name": "kanedama-backend",
-  "version": "0.0.0",
-  "private": true,
   "scripts": {
-    "prebuild": "rimraf dist",
-    "build": "nest build",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
-    "start:prod": "node dist/main",
-    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "test:cov": "jest --coverage",
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "build": "tsc --build ./tsconfig.json",
+    "start": "npm run build && node ./dist"
   },
   "dependencies": {
-    "@nestjs/common": "8.4.4",
-    "@nestjs/core": "8.4.4",
-    "@nestjs/platform-express": "8.4.4",
-    "reflect-metadata": "0.1.13",
-    "rimraf": "3.0.2",
-    "rxjs": "7.5.5"
-  },
-  "devDependencies": {
-    "@nestjs/cli": "8.2.5",
-    "@nestjs/schematics": "8.0.10",
-    "@nestjs/testing": "8.4.4",
-    "@types/express": "4.17.13",
-    "@types/jest": "27.4.1",
-    "@types/node": "16.11.27",
-    "@types/supertest": "2.0.12",
-    "@typescript-eslint/eslint-plugin": "5.21.0",
-    "@typescript-eslint/parser": "5.21.0",
-    "eslint": "8.14.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-prettier": "4.0.0",
-    "jest": "28.0.3",
-    "prettier": "2.6.2",
-    "supertest": "6.2.3",
-    "ts-jest": "27.1.4",
-    "ts-loader": "9.3.0",
-    "ts-node": "10.7.0",
-    "tsconfig-paths": "3.14.1",
-    "typescript": "4.6.4"
-  },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "ts"
-    ],
-    "rootDir": "src",
-    "testRegex": ".*\\.spec\\.ts$",
-    "transform": {
-      "^.+\\.(t|j)s$": "ts-jest"
-    },
-    "collectCoverageFrom": [
-      "**/*.(t|j)s"
-    ],
-    "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
+    "fastify": "^3.29.0",
+    "knex": "^2.0.0",
+    "@types/node": "^17.0.31",
+    "nodemon": "^2.0.16",
+    "typescript": "^4.6.4",
+    "pg": "^8.7.3"
```

I would put these modules in `devDependencies` since these are not used at runtime.

```suggestion
    "pg": "^8.7.3"
  },
  "devDependencies": {
    "@types/node": "^17.0.31",
    "nodemon": "^2.0.16",
    "typescript": "^4.6.4"
  }
```

---

yes, I introduced that in a recent commit, just to see if the candidate ships everything or has a look at that file !